lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -1132,18 +1137,10 @@ NET_DEVICE_OFFLOAD_INIT(nrf91_socket, "nrf91_socket",
 			&nrf91_iface_data, NULL,
 			0, &nrf91_iface_offload_api, 1280);
 
-#if defined(CONFIG_LTE_CONNECTIVITY)
-/* Bind l2 connectity APIs. */
-static struct conn_mgr_conn_api conn_api = {
-	.init = lte_connectivity_init,
-	.connect = lte_connectivity_connect,
-	.disconnect = lte_connectivity_disconnect,
-	.set_opt = lte_connectivity_options_set,
-	.get_opt = lte_connectivity_options_get,
-};
-
-CONN_MGR_CONN_DEFINE(LTE_CONNECTIVITY, &conn_api);
-CONN_MGR_BIND_CONN(nrf91_socket, LTE_CONNECTIVITY);
-#endif /* CONFIG_LTE_CONNECTIVITY */
+#if defined(CONFIG_NRF_MODEM_LIB_NETIF)
+extern struct conn_mgr_conn_api conn_api;
```

Comment:
I'm wondering if this struct, as included as extern, should have a more explicit name, e.g. `nrf_modem_lib_netif_conn_api `etc.?

Reply:
good point